specs/protocol/granite/predeploys.md

```diff
@@ -8,9 +8,115 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+[stage-1]: https://ethereum-magicians.org/t/proposed-milestones-for-rollups-taking-off-training-wheels/11571#stage-1-limited-training-wheels-3
+
+## Overview
+
+The OP Stack has a permissive management system for L2 predeploys that makes it
+difficult for an OP Stack chain to be considered a [Stage 1 Rollup][stage-1].
+This upgrade is meant to solve the problem of chain operators being able to
+easily break invariants important to the system by being able to arbitrarily
+modify the predeploys themselves.
+
+The `ProxyAdmin` on L2 is upgraded to the `L2ProxyAdmin`, which removes
+the code for handling the L1 legacy proxies and delineates cleanly between
+system ownership and chain operator ownership of different predeploys.
+
+## Constants
+
+| Constant          | Value                           | Description |
+| ----------------- | ------------------------------- | ----------- |
+| `PREDEPLOY_COUNT` |  2048 | The total number of predeploys on L2 |
+| `L2ProxyAdmin` address | `0x4200000000000000000000000000000000000018` | The predeploy of the `ProxyAdmin` on L2 |
+| `DEPOSITOR_ACCOUNT` | `address(0xDeaDDEaDDeAdDeAdDEAdDEaddeAddEAdDEAd0001)` | Account that represents the system |
```

Comment:
The `DEPOSITOR_ACCOUNT` would only be able to be assumed by the `op-node` under this model